### ISSUE TITLE
Can't install

### ISSUE THREAD

**Issue by NonzeroCornet** [monzelr/dmx#2]
I am unable to install using the instructions provided.

**Comment by monzelr** [comment]
Hey,this installation instruction will only install the python package "dmx".  If you get errors, please report them here. What maybe is missing, is the driver installation. You need to install the driver for the FT232R, which you can find at the product homepage (link the README.md file). Am 28.12.2023 um 05:07 schrieb NonzeroCornet34 ***@***.***>:﻿
I am unable to install using the instructions provided.

**Comment by NonzeroCornet** [comment]
Hey there! I don't think you understand. The package "dmx" will not install. I've tried upgrading and downgrading pip, going through the listed python command everything. This is what happens:

```
PS [PRIVATE]\dmx> git clone https://github.com/monzelr/dmx.git
Cloning into 'dmx'...
remote: Enumerating objects: 65, done.
remote: Counting objects: 100% (27/27), done.
remote: Compressing objects: 100% (24/24), done.
Receiving objects:  73% (48/65)used 20 (delta 3), pack-reused 38Receiving objects:  70% (46/65)
Receiving objects: 100% (65/65), 20.98 KiB | 346.00 KiB/s, done.
Resolving deltas: 100% (17/17), done.
PS [PRIVATE]\dmx>
PS [PRIVATE]\dmx> cd dmx
PS [PRIVATE]\dmx\dmx>
PS [PRIVATE]\dmx\dmx> pip install .
Defaulting to user installation because normal site-packages is not writeable
Processing [PRIVATE]\dmx\dmx
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  error: subprocess-exited-with-error

  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [18 lines of output]
      Traceback (most recent call last):
        File "[PRIVATE]\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 353, in <module>
          main()
        File "[PRIVATE]\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 335, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "[PRIVATE]\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 118, in get_requires_for_build_wheel
          return hook(config_settings)
                 ^^^^^^^^^^^^^^^^^^^^^
        File "[PRIVATE]\AppData\Local\Temp\pip-build-env-qv2ce6a2\overlay\Lib\site-packages\setuptools\build_meta.py", line 325, in get_requires_for_build_wheel
          return self._get_build_requires(config_settings, requirements=['wheel'])
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "[PRIVATE]\AppData\Local\Temp\pip-build-env-qv2ce6a2\overlay\Lib\site-packages\setuptools\build_meta.py", line 295, in _get_build_requires
          self.run_setup()
        File "[PRIVATE]\AppData\Local\Temp\pip-build-env-qv2ce6a2\overlay\Lib\site-packages\setuptools\build_meta.py", line 311, in run_setup
          exec(code, locals())
        File "<string>", line 51, in <module>
      ModuleNotFoundError: No module named 'sphinx.setup_command'
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.

[notice] A new release of pip is available: 23.3.2 -> 24.0
[notice] To update, run: [PRIVATE]\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\python.exe -m pip install --upgrade pip
```

**Comment by monzelr** [comment]
Ok, thank you for providing the code. The problem here is sphinx and its setup_command.BuildDoc which is depreciated since 7.0.0 due to setuptools.

I have pushed the fix and released the new version 0.2.3

Please re-open this issue if you still have problems installing dmx.
